packages/domain/lib/src/utils/nullable_extensions.dart

```diff
@@ -15,4 +15,5 @@ extension GetFirstIOrDefaultValue on List<String>? {
 
 extension ToIntOr on int?{
   int toIntOrPlug() => this ?? -1;
+  int toIntOrNil() => this ?? 0;
```

Comment:
naming - u shouldn't have toInt prefix. since it looks like this

int? i = 0;
i.toIntOrPlug()
i.toIntOrNil()

but i is already int. so u should have

i.orPlug()
i.orNil()

and nil is smth from the Swift, better to use orZero